morphir/sdk/core/src/org/morphir/sdk/Rule.scala

```diff
@@ -0,0 +1,29 @@
+package org.morphir.sdk
+
+import org.morphir.sdk.Bool.Bool
+import org.morphir.sdk.Maybe.Maybe
+import org.morphir.sdk.List.List
+
+object Rule {
```

Comment:
I actually plan on going back here and changing all the module names to lowercase like we discussed. So lets make this one lowercase.

Reply:
Done